@navikt/internal/react/src/timeline/Timeline.tsx

```diff
@@ -97,12 +99,20 @@ export const Timeline = forwardRef<HTMLDivElement, TimelineProps>(
     const initialEndDate = endOfDay(useLatestDate({ endDate, rows }));
     const processedRows = useTimelineRows(
       rowsRaw,
-      start,
-      endInclusive,
+      startDate ?? start,
+      endDate ?? endInclusive,
       direction
     );
 
     const handleZoomChange = (zoomStart: Date) => {
+      if (startDate || endDate) {
+        if (process.env.NODE_ENV !== "production") {
```

Comment:
would this be the first use of NODE_ENV check in our component code? (I don't see it used outside of here, but it's an internal component so it's fine?)

Reply:
Also used here: https://github.com/navikt/aksel/blob/36f4f5646a7d38cae8e6af454dd5b50966255a58/%40navikt/core/react/src/util/TextareaAutoSize.tsx#LL130C1-L130C1. 
This code is in large a copy from MUI, so they seem to be using it regularly in their system